### PR TITLE
Leaderboard - Update Stylesheet

### PR DESCRIPTION
**Pull request by BenjaminSemah**
### In this branch, I have done the following;
- Added an alert to confirm when submission is successful,
- Improved the overall look and feel of the website,
- Ensured all functionalities work as expected.
- Fixed all errors spotted by linters.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -40,7 +53,8 @@ h1 {
 }
 
 .single-score:nth-child(even) {
-  background-color: rgb(226, 224, 224);
+  /* background-color: rgb(226, 224, 224); */
```
Comment (Mihndim2020) [inline src/style.css]: - [x] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages,
:heavy_check_mark: Your app is running free of `Linter` errors,
:heavy_check_mark: Great job with UI.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (BenjaminSemah) [reply src/style.css]: Resolved. ✔️

**Comment by BenjaminSemah** [comment]
### Required Changes Implemented 
#### I have implement all required changes pointed out by previous Code Reviewer.
- Removed `commented code` from code base.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @BenjaminSemah,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**